test/init.js

```diff
@@ -21,6 +21,16 @@ try {
   };
 }
 
+if (process.env.APPVEYOR) {
```

Comment:
Now that we've got this working, should this be changed to use env vars like `MSSQL_HOST`, `MSSQL_PORT`, etc. like the other connectors do and then just set them in `appveyor.yml`?